src/applications/personalization/profile-2/components/ProfileHeader.jsx

```diff
@@ -59,7 +59,10 @@ const ProfileHeader = ({
   const titleClassesMedium = prefixUtilityClasses(['display--flex'], 'medium');
 
   const fullNameClasses = prefixUtilityClasses([
+    'font-family--serif',
```

Comment:
these classes had been applied to an `h3` element but are now being applied to a `dd` element so we need to do some additional styling to make it look right.